pkg/spiffe/spiffe_test.go

```diff
@@ -141,3 +193,98 @@ func TestGenCustomSpiffe(t *testing.T) {
 		}
 	}
 }
+
+func TestRetrieveSpiffeBundleRootCertsFromStringInput(t *testing.T) {
```

Comment:
Could you add a brief comment for the test?

Reply:
Done.